docs/tutorials/developer-journey/level-5/5.5-SNS-tutorial.md

```diff
@@ -0,0 +1,1097 @@
+
+Additionally, it is important to take into consideration that since tokenizing the dapp and placing it under control of the SNS will decentralized the control of the dapp, resulting in the development of the dapp to slow down. This is due to the process of proposing changes to the dapp via SNS proposals, requiring many people and parties to agree on the changes rather than just the dapp's developers. Therefore, it is recommended that developers of the dapp feel comfortable that code is in a stable position and that the dapp's roadmap reflects a decrease in development speed prior to handing the dapp over to the SNS.
+
+Developers should plan to continue leading the project after the launch of the SNS. This is because despite having decentralized the dapp, it is important that the original developers manage the SNS community as the project matures into its decentralization. In this stage, developers may work closely with the token holders to drive project success and direction. Developers may need to provide data regarding ecosystem growth, the project's financial changes, and take token holder input and feedback to alter the dapp's roadmap. 
+
+
+## Stages of an SNS launch
+
+- #### Step 1: Choose the initial parameters of the SNS for the dapp.
+
+- #### Step 2: Add the NNS root as co-controller of the dapp.
+
+- #### Step 3: Submit the NNS proposal to create the SNS. 
+
+- #### Step 4: Await the proposal's decision. 
+
+- #### Step 5: If the proposal is approved, the SWS-W canister deploys the SNS canisters. 
+
+- #### Step 6: The SNS-W sets the SNS root as the sole controller of the dapp. 
+
+- #### Step 7: The SNS-W initializes the SNS canisters according to the initial parameters defined in step 1. 
+
+- #### Step 8: The SNS decentralized swap begins. 
+
+- #### Step 9: The SNS decentralized swap ends. 
+
+- #### Step 10: The SNS decentralized swap finalizes. 
+
+Detailed descriptions of each step can be found in the SNS documentation [here](/docs/current/developer-docs/integrations/sns/launching/launch-summary-1proposal). This tutorial will walk through these steps below. 
+
+### SNS launch checklist
+
+Before launching an SNS, the following checklist can be used to assure that your SNS is ready for launch:
+
+- [x] Documentation:
+
+    - [x] Tokenomics specification:
+
+            - [x] Token utility: Define concise use-cases for the DAO's token.
+
+            - [x] Initial token allocation amounts, including which groups or accounts receive how many tokens.  
+
+            - [x] Voting power: Provide information about how voting power will be distributed at the SNS's genesis.
+
+            - [x] Funding target: Maximum and minimum targets needed for the decentralized swap, including information about planned usage of raised funds. 
+
+            - [x] Use the [SNS tokenomics tool](https://docs.google.com/spreadsheets/u/0/d/1eSxkJl94jPt63CdOXH6ROy-WSkacW6P4qcAKMLrfBPc/edit) to analyze and document the SNS's proposed tokenomics. 
```

Comment:
@bjoernek Do we still want to advertise the SNS tokenomics tool?

Reply:
I think we should (but we should refer to the new tool on Github).

Reply:
Resolving this since the SNS tutorial has been removed per my conversation with @LaraAS